lib/Weathermap/Integrations/Cacti/WeatherMapCactiManagementPlugin.php

```diff
@@ -626,6 +658,23 @@ function handleMapListAdd(
         }
     }
 
+
+    protected
+    function handleMapsListAdd(
```

Comment:
It may be better to stick to the spelling standard to write all in one line:
`protected function handleMapsListAdd(`

Reply:
I think that was a mistake.  Howie doesn't normally have lines like that.  Good spot!

Reply:
Should be done now.